models/issues/tracked_time.go

```diff
@@ -251,10 +254,12 @@ func DeleteIssueUserTimes(issue *Issue, user *user_model.User) error {
 		return err
 	}
 	if _, err := CreateComment(ctx, &CreateCommentOptions{
-		Issue:   issue,
-		Repo:    issue.Repo,
-		Doer:    user,
-		Content: "- " + util.SecToTime(removedTime),
+		Issue: issue,
+		Repo:  issue.Repo,
+		Doer:  user,
+		// Content before v1.21 did store the formated string instead of seconds,
+		// the legacy format starts with '- '
+		Content: fmt.Sprint(removedTime),
```

Comment:
Above is `Content: fmt.Sprintf("|%d", amount),` here is `Content: fmt.Sprint(removedTime)`

Could there be a unified format for the "seconds"? eg: all have the prefix `|` ?

Reply:
done